app/components/request_form/request_form.css

```diff
@@ -10,6 +10,20 @@
   top: calc(3.5 * var(--spacing-unit));
 }
 
+.RequestForm-messageTextAreaToolbar {
+  width: 100%;
+  border: 2px solid transparent;
```

Comment:
So, to be honest, this was a copy/:spaghetti: from [Toolbar Example](https://www.w3.org/WAI/ARIA/apg/example-index/toolbar/toolbar.html), which illustrates the[ roving tabindex method](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_roving_tabindex). It is originally `transparent`, but the `border-color` is set on focus so that when one is navigating the page with tabs, it is clear where they are. You also jump to the first item in the toolbar with tab and move to the next element outside the toolbar with tab. If you want to use your keyboard to navigate the toolbar itself, you would then switch to the right/left arrows.

One, I did not set up the focus styles, which I should probably do since it is part of `a11y`. Two, I don't think we really want to add the complexity that comes with the `roving tabindex method` at this point. Especially since we only have one item in the toolbar, but it's something to keep in mind.

[Keyboard Accessibility](https://webaim.org/techniques/keyboard/)I guess is important for us to keep in mind while building our site, as it:

>  is one of the most important aspects of web accessibility.

Reply:
Hi, I see where that came from and I agree that keyboard accessibility is important and just plainly improves usability. 
I am attaching two screenshots of navigating our form via keyboard (the screenshots are one tab away). In our case the button within the toolbar gets into focus immediately instead of first focusing on the toolbar. Works well for me and I agree that the `roving tabindex method` currently appears overkill, even though its a good thing to keep in the back of our minds! I think the transparent border of the toolbar element, therefore, isn't needed currently but I also agree with you that we should set the `focus` styles of the button to match the input elements `focus` styles. I'll add some suggestions.

<img width="383" alt="image" src="https://user-images.githubusercontent.com/34538290/189303950-3672d31b-9c36-4fa3-9a4d-5d33cff609ac.png">
<img width="465" alt="image" src="https://user-images.githubusercontent.com/34538290/189303997-c81b55b1-d1a5-4faa-83c2-7001c198a205.png">

Reply:
Addressed this in a comment :point_down: